### PR TITLE
fixed: 404 redirects

### PR DESCRIPTION
**Pull request by dpshekhawat**
**PR  fix -**
- broken link on the webpage ([why_unify.html](https://lets-unify.ai/ivy/background/why_unify.html))

### CODE REVIEW

File: docs/partial_source/background/why_unify.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ With Ivy, this process becomes:
 
 We have turned a 4-step process which can take 100s of hours into a 1-step process which takes a few seconds.
 
-.. image:: https://github.com/unifyai/unifyai.github.io/blob/master/img/externally_linked/why_unify/perceiver_effort.png?raw=true
+.. image:: https://github.com/unifyai/unifyai.github.io/blob/master/img/externally_linked/background/why_unify/perceiver_effort.png
```
Comment (VedPatwardhan) [inline docs/partial_source/background/why_unify.rst:31]: Please add `?raw=true` at the end of the link.

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]
Thanks for requesting the review @Ishticode. Hi @dpshekhawat, just suggested a minor change, @Ishticode please feel free to merge the PR once it's done. Thanks 🙂

File: docs/partial_source/background/why_unify.rst (hunk 1 above)
Comment (dpshekhawat) [reply docs/partial_source/background/why_unify.rst]: Done!

**Review by Ishticode — APPROVED** [review]
@dpshekhawat Looks good. Should get it merged. Thank you very much for the contribution. 🙂

**Pull request merged**